WcaOnRails/db/structure.sql

```diff
@@ -746,7 +747,7 @@ CREATE TABLE `competition_events` (
   `competition_id` varchar(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
   `event_id` varchar(191) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
   `fee_lowest_denomination` int(11) NOT NULL DEFAULT '0',
-  `qualification` text CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci,
```

Comment:
Unrelated change

Reply:
Argh, every time I edit the db schema unrelated changes creep in...

Fixed.